### ISSUE TITLE
change made between OS 3.3 and 3.4 RC1 lose Solar Absoprtance import from gbXML

### ISSUE THREAD

**Issue by DavidGoldwasser** [NREL/OpenStudio#4569]
There were some commits after 3.3 that changed gbXML to fix UUIDs being written as names. Note that now ID string and not name string are used for objects (not sure if that was the intent)

But this bug is about something else. While gbXML materials don't have solar absorptance inputs, the constructions do. In OpenStudio 3.3 and earlier these were found in the gbXML and imported but this doesn't happen in 3.4. This resulted in 13% reduction of energy use in the 3.4 test model vs. the 3.3 test model. Screenshot below has 3.3 at top left, 3.4 at bottom left, and gbXML at Right (with material selected). Note material 046 is in Layer 045, which is in Construction 041.

Here is construction with absorptance value
` <Construction id="aim0041">
    <U-value unit="WPerSquareMeterK">3.4317585</U-value>
    <Absorptance unit="Fraction" type="ExtIR">0.1</Absorptance>
    <Roughness value="VeryRough" />
    <LayerId layerIdRef="aim0045" />
    <Name>Floor: Generic - 12"</Name>
  </Construction>`

![Screen Shot 2022-04-15 at 10 56 09 AM](https://user-images.githubusercontent.com/4857055/163598780-94af4d3e-c646-4fab-909b-5938cb4cb152.png)

cc @axelstudios @tijcolem
.

**Comment by tijcolem** [comment]
@DavidGoldwasser Could you send over files to @joseph-robertson to reproduce?

**Comment by DavidGoldwasser** [comment]
Closing this as it isn't directly related to changes in OpenStudio. OpenStudio doesn't actually extract solar absoprtance data from the gbXML file. A separate measure was making a second pass through the gbXML to extract absoprtance from construction in gbXML and assign it to exterior material in OpenStudio. This riles on the expected construction name in OpenStudio matching the name string for the construction in the gbXML. Between 3.3 and 3.4 OpenStudio's gbXML reverse translator changed to using the id string from the construction object instead of the name attribute for the OpenStudio object name. This is what caused the change in the resulting model. I was able to update the measure to operate different pre/post 3.4. I suggest that for 3.5 or some future release at least some of the elements from the advanced gbxml import get brought into OpenStudio's gbXML reverse translators.
